Components/modal/modal.css

```diff
@@ -0,0 +1,40 @@
+/* modal */
+
+.modal-container.active {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(0,0,0,0.5);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.modal-content {
+    position: relative;
+}
+
+.modal-container {
+    display: none;
+    z-index: 3;
+    padding: 0 1rem;
+}
```

Comment:
this class can go on top of `.active`. do you think it would be better if the base styles of a class should sit on the top and all its variations or states can come below in the source order?

Reply:
thanks for that. I will change the order.